tests/unit/test_loader.py

```diff
@@ -29,6 +29,12 @@
 
 class LoaderTestCase(unittest.TestCase):
 
+    def setUp(self):
+        ApacheConfigLexer = make_lexer()
+        ApacheConfigParser = make_parser()
+        self._loader = ApacheConfigLoader(
+            ApacheConfigParser(ApacheConfigLexer()))
+
```

Comment:
Eventually, we might want to have the tests re-use the same loader instead of recreating it! Same with the parser objects in `test_parser.py`.

Reply:
Why re-using these objects would be beneficial? I thought `setUp` recreates everything from the scratch to minimize interference between tests, no?

Reply:
Nothing more than to reduce code re-use :)